qiskit/algorithms/evolvers/real/trotterization/__init__.py

```diff
@@ -0,0 +1,22 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""This package contains Trotterization-based Quantum Real Time Evolution algorithm.
+It is compliant with the new Quantum Time Evolution Framework and makes use of `ProductFormula` and
```

Comment:
I think these references to other classes e.g. ProductFormula could be done as proper Sphinx links, assuming this shows up in the docs, as it would allow the user to navigate there simply to see the class